### PR TITLE
fix api issue and add response page for create

### PR DESCRIPTION
**Pull request by mahendra-egov**

### CODE REVIEW

File: packages/modules/pt/src/pageComponents/PropertyFloorDetails.js [hunk 1]
```diff
@@ -34,6 +34,23 @@ const PropertyFloorsDetails = ({ t, config, onSelect, formData, userType }) => {
     },
   ];
 
+  const employeeMenu = [
+    {
```
Comment (jagankumar-egov) [inline packages/modules/pt/src/pageComponents/PropertyFloorDetails.js:38]: For Employee side no of floors is not fixed , it should be fetched from mdms 
only on citizen floor no is fixed

PropertyTax: -> 
Floor

File: packages/modules/pt/src/pages/employee/AssessmentDetails.js [hunk 2]
```diff
@@ -68,7 +68,7 @@ const AssessmentDetails = () => {
       { Assessment: AssessmentData },
       {
         onError: (error, variables) => {
-          setShowToast({ key: "error", action: error });
+          setShowToast({ key: "error", action: error?.response?.data?.Errors[0]?.message || error });
```
Comment (jagankumar-egov) [inline packages/modules/pt/src/pages/employee/AssessmentDetails.js:71]: while creating assessment from employee ,  adhoc penalty and rebate option should be provided

**Review by jagankumar-egov — CHANGES_REQUESTED** [review]
@mahendra-egov  can you check those 2 comments?

File: packages/modules/pt/src/pageComponents/PropertyFloorDetails.js (hunk 1 above)
Comment (mahendra-egov) [reply packages/modules/pt/src/pageComponents/PropertyFloorDetails.js]: done

File: packages/modules/pt/src/pages/employee/AssessmentDetails.js (hunk 2 above)
Comment (mahendra-egov) [reply packages/modules/pt/src/pages/employee/AssessmentDetails.js:71]: I will start working on it on new branch once Ajit confirm

**Review by jagankumar-egov — APPROVED** [review]

**Pull request merged**